src/plugins/router/index.ts

```diff
@@ -205,8 +213,14 @@ if (isGoerli.value) {
 const router = createRouter({
   history: createWebHashHistory(),
   routes,
-  scrollBehavior() {
-    return { top: 0 };
+  scrollBehavior(to, from, savedPosition) {
```

Comment:
Please review these changes in detail. I did this to ensure that anchor links would work (previously, anchor links would place users at the top of the page). 

One nice advantage is that people return to the position they were previously at. So if a user on the home page loads some extra rows and scrolls down before clicking in to a pool, they will be returned to the same position if they return back to the homepage. 

Could this have any other unintended consequences?

Reply:
hm, I think it's ok, but need some testing